### PR TITLE
release: 1.0.0

### PR DESCRIPTION
**Pull request by qubitrenegade**
## Summary

First stable release of clickwork. Cuts v1.0.0 with:

- `version = \"1.0.0\"` in pyproject.toml
- Trove classifier bumped to `Development Status :: 5 - Production/Stable`
- CHANGELOG.md entry documenting every breaking change, addition, and CI tightening since 0.2.0

## 1.0 scope (24 issues closed)

- **Breaking**: env-sourced config coercion (#41), logger host-preservation (#43), fail-fast undefined env (#52), Python 3.11+ floor (#46), Click 8.2+ pin (#46, PR #33).
- **Public API policy**: docs/API_POLICY.md with semver commitments + deprecation runway (#36, #46, #49).
- **Docs**: MIGRATING.md (#56), PLUGINS.md (#53), SECURITY.md (#55), CONTRIBUTING.md (#58), GUIDE.md Config Precedence table (#54), API_POLICY.md (#36 etc.).
- **Features**: strict discovery (#42), --version flag (#48), _deprecated decorator (#47), py.typed marker (#37), add_global_option docs (#60).
- **CI**: strict mypy (#38), ruff 0.6.9 (#38), cross-platform matrix (#39), wheel/sdist smoke (#40), cold-start benchmark (#59), auto release notes (#57).

## Validation

- 336/336 tests pass locally on main + this bump.
- orbit-admin runtime validated against 1.0 RC — clean (#44).
- All Wave 1–4 PRs (19 PRs) through Copilot review cycles and merged.

## Test plan

- [x] `pytest` green on release branch
- [x] orbit-admin runtime sanity-checked against 1.0 RC
- [ ] CI (mypy, ruff, multi-OS, click-edge) green on PR
- [ ] Wheel + sdist smoke passes
- [ ] User approves release cut — tagging + GitHub release triggers PyPI trusted-publisher flow

After merge: tag `v1.0.0`, `gh release create v1.0.0` to trigger publish.yml, then approve the `pypi` environment deployment.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [project]
 name = "clickwork"
-version = "0.2.0"
+version = "1.0.0"
```
[the remark this answers is not in the document]
Comment (qubitrenegade) [reply pyproject.toml:3]: Fixed in bb56bfc — bumped `__version__` to 1.0.0 in src/clickwork/__init__.py. Good catch, this would have broken any consumer's `--version` resolution.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,133 @@ All notable changes to clickwork will be documented in this file.
...
+  env and listing the defined ones. Previously this silently
+  returned the `[default]` merge. Missing repo config file is still
+  a no-op. (#52)
+- **Python floor bumped to 3.11+.** This is a hard floor:
+  `tomllib` is now imported unconditionally from the standard
+  library. Consumers on 3.9/3.10 must upgrade Python. (#46)
+- **Click floor pinned to `>=8.2,<9`.** 0.2.0 already required
+  `click>=8.2`; 1.0 makes this explicit in metadata and caps at `<9`
+  so a future Click major doesn't break consumers without a
+  clickwork release. (#46, PR #33)
```
[the remark this answers is not in the document]
Comment (qubitrenegade) [reply CHANGELOG.md]: Fixed in bb56bfc — I'd inverted the policy claim. The actual decision (API_POLICY.md 'Click version range') is no upper bound, and the CHANGELOG now reflects that with the rationale.

File: src/clickwork/__init__.py [hunk 3]
```diff
@@ -25,7 +25,7 @@
     testing           - Submodule exposing run_cli() + make_test_cli() helpers
 """
 
-__version__ = "0.2.0"
+__version__ = "1.0.0"
```
[the remark this answers is not in the document]
Comment (qubitrenegade) [reply src/clickwork/__init__.py]: Fixed in 3ca869a — `__version__` now derives from `importlib.metadata.version("clickwork")` so pyproject.toml is the single source of truth and the two can't drift again. PackageNotFoundError fallback handles the uninstalled-checkout edge case. Verified locally: `clickwork.__version__` reports 1.0.0 matching pyproject, 331 unit tests pass, mypy --strict clean, ruff clean.
